### DISCUSSION TITLE
ConBee II vs  CC2652/CC1352 chip-based adapters?

### DISCUSSION THREAD

**Discussion by flemmingss** [Koenkk/zigbee2mqtt#10291] [Devices]
Ref https://www.zigbee2mqtt.io/guide/adapters/#recommended

I hav a ConBee II connected myself, and the network seems to be stable and good.
I just have problems with:
- ZigBee HAN-adapters like [https://www.zigbee2mqtt.io/devices/HSE2905E.html](https://www.zigbee2mqtt.io/devices/HSE2905E.html) (no data) and [this](https://www.zigbee2mqtt.io/devices/EMIZB-132.html) (wrong data).
- Unable to use Touchlink

Will a new ordinater fix hese problems? And how is ConBee II in generel compaird to the rest? I guess it should be best becaus it cost a lot mor then almost every other.
I i decide to change, can i migrate my network to another one ?

**Comment by alex-v-fraser** [comment]
I have raspbee II - its network is good, stable with 250-255lqi. Yesterday I replaced it with Sonoff zigbee 3.0 usb dongle and now I have 5-100lqi on all devices in network. I want Conbee II ( I think it has the same scheme as Raspbee II) to get a good quality of network signal

**Reply by Koenkk** [reply]
The LQI on the Conbee is bugged (firmware bug of the Conbee)
